arwen/host/execution.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/ElrondNetwork/elrond-go/core/vmcommon"
 )
 
+const gasLimitToUseForRevertESDT = uint64(1_500_000_000)
```

Comment:
why hardcoding this here?

Reply:
where should I hardcode it ?

Reply:
do we pass to arwen the maximum gas limit per miniblock?

Reply:
I see that in host.go we have some consts defined